src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -142,22 +156,69 @@ class TransportCcEngine(
         }
     }
 
+    /** Trim old sent packet details from the map.  Caller should be synchronized on [sentPacketsSyncRoot]. */
```

Comment:
Can you document what the return value indicates?

Reply:
Ok.